jetbrains-core/src/software/aws/toolkits/jetbrains/ui/feedback/FeedbackDialog.kt

```diff
@@ -32,19 +39,31 @@ class FeedbackDialog(project: Project) : DialogWrapper(project) {
 
             val sentiment = panel.sentiment ?: throw IllegalStateException("sentiment was null after validation")
             val comment = panel.comment ?: throw IllegalStateException("comment was null after validation")
-            ApplicationManager.getApplication().executeOnPooledThread {
+            launch(coroutineContext) {
+                val edtDispatcher = AppUIExecutor.onUiThread(ModalityState.stateForComponent(panel.panel)).coroutineDispatchingContext()
```

Comment:
I made a function for this in the branch I'm working out of now, it also includes a disposable if the component is disposed:
```
fun getCoroutineUiContext(modalityState: ModalityState = ModalityState.any(), disposable: Disposable? = null): CoroutineContext {
    val uiThread = AppUIExecutor.onUiThread(modalityState)
    return if (disposable == null) {
        uiThread
    } else {
        uiThread.expireWith(disposable)
    }.coroutineDispatchingContext()
}
```